decimal64math.go

```diff
@@ -7,41 +7,76 @@ func (d Decimal64) Abs() Decimal64 {
 
 // Add computes d + e
 func (d Decimal64) Add(e Decimal64) Decimal64 {
-	flavor1, sign1, exp1, significand1 := d.parts()
-	flavor2, sign2, exp2, significand2 := e.parts()
-	if flavor1 == flSNaN || flavor2 == flSNaN {
+	dDec := d.decParts()
+	eDec := e.decParts()
```

Comment:
nit: `dec` is the adjective of `decParts`. Better to call them `dParts` and `eParts` (or just `ep` and `dp`, given their narrow scope).